scripts/set_modes.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
what does this script do?

Reply:
added comment "# this script is using one during installation to create /public_html and /domlogs folders and set access modes"